module/VuFind/src/VuFind/View/Helper/Root/RecordDataFormatter.php

```diff
@@ -63,6 +63,68 @@ public function specSortCallback($a, $b)
         return ($a['pos'] ?? 0) <=> ($b['pos'] ?? 0);
     }
 
+    /**
+     * Should we allow a value? (Performs zero checking on a value that
+     * evaluates to empty).
```

Comment:
"..that doesn't evaluate to empty", right?

Reply:
Oh no, I read it wrong... I read "performs zero checking" as "performs no checking". Maybe revise the wording a bit?

Reply:
I've revised the wording; hopefully it makes more sense now.